code/online_challenges/src/hackerrank/2D Array - DS

```diff
@@ -0,0 +1,59 @@
+import java.io.*;
```

Comment:
Create a folder for this problem and make sure there are no spaces in it. Also use underscores for separators. This file is also missing a file extension.